tests/fixtures/user-batchupdate-event-payload.json

```diff
@@ -0,0 +1,314 @@
+{
+    "notification_id": "cc156fd4-5150-4f1f-908f-63287beafe5d",
+    "configuration": {
+        "id": "5bbb8592ec9ecaa580002cff",
+        "organization": "ad793dc7.hullapp.io",
```

Comment:
I would avoid using references to organizations that exist in the tests. We can use `xxx.hullapp.dev` domains as a convention instead of `xxx.hullapp.io` domains

Reply:
yea, makes sense